Parse.xcodeproj/project.pbxproj

```diff
@@ -5894,6 +5947,110 @@
 			};
 			name = Release;
 		};
+		4A26C3BD1C053C2D00E5F505 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = 4A26C3B61C053AEE00E5F505 /* Parse-iOS-Dynamic.xcconfig */;
```

Comment:
Can we remove all these build settings from here? 
xcconfig files are exactly for this purpose, since it's crazy to merge these.